src/js/tracks/text-track-display.js

```diff
@@ -103,7 +121,20 @@ class TextTrackDisplay extends Component {
       cues.push(track['activeCues'][i]);
     }
 
-    window['WebVTT']['processCues'](window, track['activeCues'], this.el_);
+    if (track['kind'] === 'descriptions') {
+
+      if ((overrides.descriptionsPlayback) && (overrides.descriptionsPlayback === 'screenReaderOnly')) {
```

Comment:
The way we have it implemented right now, we don't support having multiple tracks showing.
A solution would be to just merge all the active cues we want to display into one array and give that to vttjs. But this would require some refactor.